### DISCUSSION TITLE
Suggestion: Change to the order of plug-ins on the side of the Runelite banner

### DISCUSSION THREAD

**Discussion by Multiman20** [runelite/runelite#18006] [Ideas]
Hey there. After having some trouble rearranging the order of the plug-ins on the side of my runelite tab (xp tracker, grand exchange etc.) I was wondering if it would be possible to create something so as to rearrange the order of those helpful plugins to prioritize and optimize skilling and playstyle. This is primarily a suggestion to have the ability to change the order of the core icons on the side while not getting rid of any of the existing ones. 

That is all. Thank you.

**Comment by cdfisher** [comment]
Duplicate of #15020 (among others)
